pkg/standalone-utils/test/BaseRelayerLibrary.test.ts

```diff
@@ -294,6 +294,19 @@ describe('BaseRelayerLibrary', function () {
         });
       });
     });
+
+    describe('peekChainedReferenceValue', () => {
+      it('peeks chained reference', async () => {
+        const reference = toChainedReference(174);
+        const value = fp(340);
+
+        const result = await relayer.callStatic.multicall([
+          relayerLibrary.interface.encodeFunctionData('setChainedReferenceValue', [reference, value]),
+          relayerLibrary.interface.encodeFunctionData('peekChainedReferenceValue', [reference]),
+        ]);
+        expect(result).to.be.deep.eq([bn(0), value]);
```

Comment:
Heh, this made me realize we have a small bug in `setChainedRef`: that function should not return anything.

Reply:
Changed in both mock relayer library implementations, and adjusted test.